src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private double firstLeg, secondLeg, thirdLeg;
+    private String color;
+
+    public RightTriangle(double firstLeg, String color) {
+        this.firstLeg = firstLeg;
+        this.secondLeg = firstLeg;
+        this.thirdLeg = firstLeg;
+        this.color = color;
+    }
+
+    public RightTriangle() {
+    }
```

Comment:
If the empty constructor is not used, it's better to remove it to keep the code clean.